locale.js

```diff
@@ -0,0 +1,67 @@
+const fs = require("fs");
+const path = require("path");
+const localePath = path.join(require.main.path, "lang");
+const languageFolders = fs.readdirSync(localePath);
+
+const languages = {};
+const CURRENT_LANGUAGE = "es_ar";

```

Comment:
```suggestion
const CURRENT_LANGUAGE = "en";
```
Remember to default this to "en" as a default before uploading changes.